industry_benchmarks/utils/tests/test_traj_cleanup.py

```diff
@@ -0,0 +1,61 @@
+import MDAnalysis as mda
+import pytest
+from importlib import resources
+from ..traj_cleanup import extract_data
+import numpy as np
+
+
+@pytest.fixture
+def simulation():
+    with resources.files('utils.tests.data.example_traj') as d:
+        yield d / 'simulation.nc'
+
+
+@pytest.fixture
+def checkpoint():
+    with resources.files('utils.tests.data.example_traj') as d:
+        yield d / 'checkpoint.chk'
+
+
+@pytest.fixture
+def pdb():
+    with resources.files('utils.tests.data.example_traj') as d:
+        yield d / 'hybrid_system.pdb'
+
+
+@pytest.fixture
+def outfile():
+    with resources.files('utils.tests.data.example_traj') as d:
+        yield d / 'data.npz'
+
+
+@pytest.fixture
+def out_traj():
+    with resources.files('utils.tests.data.example_traj') as d:
+        yield d / 'out'
+
+
+def test_extract_data(simulation, checkpoint, pdb, outfile, out_traj):
+
+    extract_data(simulation, checkpoint, pdb, outfile, out_traj)
```

Comment:
Should this be wrapped around a tempdir call to avoid re-writing / storing files?